docs/docs/linking-to-a-route.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Linking to a route
+sidebar_label: Linking to a route
+---
+
+Now that we have created our router, let's create our first link to our route:
+
+```tsx {8,9} title="src/Header.tsx"
```

Comment:
Is this line correct?

Reply:
I guess? What do you mean?

Reply:
I didn't checked by running the doc, I don't know what this {8, 9} means

Reply:
It just highlights the lines 8 and 9

Reply:
Oh, ok